app/src/components/common/layout/footer/index.tsx

```diff
@@ -40,14 +39,12 @@ const Break = styled.span`
 `
 
 export const Footer = () => {
-  const context = useConnectedWeb3Context()
-  const { marketMakerFactory } = useContracts(context)
   return SHOW_FOOTER ? (
     <>
       <Wrapper paddingBottomSmall={DISCLAIMER_TEXT ? true : false}>
         <Link href="https://github.com/protofire/gnosis-conditional-exchange">Version {appVersion}</Link>
         <Break>·</Break>
-        <Link href={`https://etherscan.io/address/${marketMakerFactory.address}`}>Omen Contract</Link>
+        <Link href="https://etherscan.io/address/0x89023DEb1d9a9a62fF3A5ca8F23Be8d87A576220">Omen Contract</Link>
```

Comment:
The contract address should be in the `constants` file

Reply:
Added it as `MARKET_MAKER_FACTORY_ADDRESS`